tests/conftest.py

```diff
@@ -259,6 +259,9 @@ def _make_package(name, version="0.1", install_requires=None):
                     setup(
                         name={name!r},
                         version={version!r},
+                        author="pip-tools",
+                        author_email="pip-tools@mailinator.com",
```

Comment:
```suggestion
                        author_email="pip-tools@localhost",
```

Maybe localhost to avoid ads?